gridscale/resource_gridscale_server.go

```diff
@@ -191,6 +191,7 @@ func resourceGridscaleServer() *schema.Resource {
 						"ordering": {
 							Type:       schema.TypeInt,
 							Optional:   true,
```

Comment:
Do we still need the `Optional` attribute with `Computed`?

Reply:
> Do we still need the `Optional` attribute with `Computed`?

Yes. If you remove the `optional` attribute, the backward compatibility is broken (if some users already set that field `ordering`)